common/tier4_autoware_utils/CMakeLists.txt

```diff
@@ -19,6 +19,13 @@ ament_auto_add_library(tier4_autoware_utils SHARED
   src/planning/planning_marker_helper.cpp
 )
 
+# workaround to allow deprecated header to build on both galactic and rolling
+if(${tf2_geometry_msgs_VERSION} VERSION_LESS 0.18.0)
```

Comment:
@esteve I'm not willing to write version compare macro, so I compare version in cmake. Is it ok for you?

Reply:
Sounds good to me :+1: